### PR TITLE
Add other record types to domain

### PR DESCRIPTION
**Pull request by ajdelgado**

### CODE REVIEW

**Review by Boulet- — APPROVED** [review]

File: plugins/modules/domain.py [hunk 1]
```diff
@@ -55,9 +58,10 @@
 def run_module():
     module_args = ovh_argument_spec()
     module_args.update(dict(
-        ip=dict(required=True),
+        value=dict(required=True),
         name=dict(required=True),
         domain=dict(required=True),
+        record_type=dict(choices=['A', 'CNAME', 'TXT', 'AAA', 'NS', 'SRV', 'MX'], default='A'),
```
Comment (Skunnyk) [inline plugins/modules/domain.py:64]: `'AAAA'`

File: README.md [hunk 2]
```diff
@@ -106,7 +106,7 @@ A few examples:
 - name: Add server IP to DNS
   synthesio.ovh.domain:
     domain: "example.com"
-    ip: "192.0.2.1"
+    value: "192.0.2.1"
     name: "internal.bar"
```
Comment (Skunnyk) [inline README.md:110]: pleazse add "record_type" in example

File: plugins/modules/domain.py [hunk 3]
```diff
@@ -17,15 +17,18 @@
...
+        description: The value of the record
     name:
         required: true
         description: The name to create/update/delete
     domain:
         required: true
         description: The domain to modify
+    record_type:
+        required: false
+        description: The DNS record type (A, CNAME, TXT, AAA, NS, SRV, MX)
```
Comment (Skunnyk) [inline plugins/modules/domain.py:31]: `AAAA`

Comment (ajdelgado) [reply plugins/modules/domain.py]: Done, thanks!

File: README.md (hunk 2 above)
Comment (ajdelgado) [reply README.md:111]: Done!

File: plugins/modules/domain.py (hunk 1 above)
Comment (ajdelgado) [reply plugins/modules/domain.py]: Done

**Comment by Skunnyk** [comment]
Thank you ! :)

**Pull request merged**